catalog/spatial/kml/spatial-kml-transformer/src/main/java/org/codice/ddf/spatial/kml/transformer/KMLTransformerImpl.java

```diff
@@ -137,34 +111,16 @@ public KMLTransformerImpl(
     this.styleMapper = mapper;
     this.templateHelper = new DescriptionTemplateHelper(actionProvider);
 
-    URL stylingUrl = context.getBundle().getResource(defaultStylingName);
+    final URL stylingUrl = context.getBundle().getResource(defaultStylingName);
```

Comment:
:question:   Is `null` check on `context`?

Reply:
I don't think we null check constructor parameters.

Reply:
why not?

Reply:
Added.